### PR TITLE
Allow absolute paths in `extends` property of configs

### PR DESCRIPTION
**Pull request by SilverFire**
This pull requests adds support of absolute paths in `extends` property on configs and adds tests that the new feature works.

### CODE REVIEW

File: CHANGELOG-2.4.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * [Asserts] Added new methods `assertNotTrue` and `assertNotFalse` methods; by @johannesschobel
 * [REST][PhpBrowser][Frameworks] Added new methods to check for `Http Status Ranges` with nice "wrappers" (e.g., `seeHttpStatusCodeIsSuccessful()` checks the code between 200 and 299); by @johannesschobel
 * Improved the docs; by community
+* Recently added `extends` property in the `codeception.yml` and `*.suite.yml` files now support absolute paths; by @silverfire
```
Comment (DavertMik) [inline CHANGELOG-2.4.md]: Should go to 2.4.4

File: src/Codeception/Configuration.php [hunk 2]
```diff
@@ -690,7 +690,10 @@ protected static function loadSuiteConfig($suite, $path, $settings)
 
         // now we check the suite config file, if a extends key is defined
         if (isset($suiteConf['extends'])) {
-            $presetFilePath = realpath($suiteDir . DIRECTORY_SEPARATOR . $suiteConf['extends']);
+            $presetFilePath = mb_substr($suiteConf['extends'], 0, 1) === DIRECTORY_SEPARATOR
```
Comment (DavertMik) [inline src/Codeception/Configuration.php]: how about windows paths???

File: tests/cli/ConfigExtendsCest.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+{
+    /**
+     * @param CliGuy $I
+     */
+    public function runIncludedSuites(\CliGuy $I)
+    {
+        $I->amInPath('tests/data/config_extends');
+        $I->executeCommand('run');
+
+        $I->seeInShellOutput('✔');
```
Comment (DavertMik) [inline tests/cli/ConfigExtendsCest.php]: on Windows this character is not displayed (by default). Check for something else.
See Appveyor failing build

File: src/Codeception/Configuration.php (hunk 2 above)
Comment (SilverFire) [reply src/Codeception/Configuration.php]: It's called discrimination :)

File: CHANGELOG-2.4.md (hunk 1 above)
Comment (SilverFire) [reply CHANGELOG-2.4.md]: +

**Comment by SilverFire** [comment]
Ready for merge

**Pull request merged**